agnostic-svelte/src/lib/components/Table/Table.stories.js

```diff
@@ -138,3 +138,9 @@ CaptionBottom.args = { ...defaultArgs, captionPosition: "bottom" };
 
 export const CaptionEnd = Template.bind({});
 CaptionEnd.args = { ...defaultArgs, captionPosition: "end" };
+
+export const LimitFive = Template.bind({});
+LimitFive.args = { ...defaultArgs, limit: 5 };
+
+export const OffsetFive = Template.bind({});
```

Comment:
I think there's 8 rows in the fixture right? So maybe we should have a test with both an offset of 2 and a limit of 3 or something just to test using both props with the size of dataset?

Reply:
Alright added an Offset2Limit3 spec